index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Ernest-Kell Tribute WebPage</title>
+</head>
+<body>
+  <main id="main">
+  <h1 id="title">Dr Ernest Kell</h1>
+  <div id="img">
+    <img id="image" src="EK.jpeg" alt="Dr Ernest Kell is Elon musks dad"/>
+   <figcaption id="img-caption" > Dr Ernest Kell is a renowned alien, who wins at life, because code matters.  </figcaption>
+    </div>
+
+<section id="Tribute-info">
+    <h3>Here is a time line of Dr Ernest Kell's life!</h3>
+</section id ="headline">
+
+<ul>
+    <li>
+        <strong>1942</strong> WAVES BORN: Baby Dr Ernest Kell born via waves.
+    </li>
+    <li>
+        <strong>1998</strong> WAVES SURFED; Grown Dr spends time surfing waves.
+    </li>
+    <li>
+      <strong>2024</strong> WAVES CREATED: Wise Dr expands waves via creation.
+    </li>
+    <li>
+      <strong>2042</strong> WAVES LEARNT: Dr learns to become the wave. 
+    </li>
+    <li>
+        <strong>2099</strong> WAVES ENJOYED: Dr sits and enjoys all waves.
+    </li>
+</ul>
+
+      <!--Make sure to clean up code with un-necessary indentations-->
```

Comment:
- [ ] Kindly write this as an inline comment, not a comment as shown in the video below. "Make sure to clean up code with unnecessary indentations".

https://user-images.githubusercontent.com/70269507/165059611-423c881c-a5db-42c5-93fb-f34ea530542a.mov

Reply:
Ok @sja-thedude and @m05e5 I will do it now!